app/main.py

```diff
@@ -2,5 +2,16 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    closure_list = {}
```

Comment:
is it list ?)
it is better to name vars according to their types to avoid misunderstands